public/components/Flyout/flyouts/createDetector.js

```diff
@@ -0,0 +1,814 @@
+                            context.suggestedChanges,
+                            'filter_query'
+                          )}
+                        </EuiFlexItem>
+                      </EuiFlexGroup>
+                    </Fragment>
+                    <EuiFlexGroup
+                      alignItems="center"
+                      justifyContent="flexEnd"
+                      style={{ paddingTop: '10px' }}
+                    >
+                      <EuiFlexItem grow={false}>
+                        <EuiButton color="secondary" onClick={handleSubmit}>
+                          Validate
+                        </EuiButton>
+                      </EuiFlexItem>
+                    </EuiFlexGroup>
+                  </Fragment>
+                )}
+              />
+            ) : (
+              <EuiFlexGrid columns={2} gutterSize="l" style={{ border: 'none' }}>
+                <EuiFlexItem>
+                  <ConfigCell title="Name" description={context.adConfigs.name} />
+                </EuiFlexItem>
+                <EuiFlexItem>
+                  <ConfigCell title="Description" description={context.adConfigs.description} />
+                </EuiFlexItem>
+                <EuiFlexItem>
+                  <ConfigCell title="Data source index" description={context.adConfigs.indices} />
+                </EuiFlexItem>
+                <EuiFlexItem>
+                  <ConfigCell
+                    title="Detector interval"
+                    description={toString(context.adConfigs.detection_interval)}
+                  />
+                </EuiFlexItem>
+                <EuiFlexItem>
+                  <ConfigCell
+                    title="Window delay"
+                    description={toString(context.adConfigs.window_delay)}
+                  />
+                </EuiFlexItem>
+                <EuiFlexItem>
+                  <ConfigCell
+                    title="Data filter"
+                    description={displayText(context.queriesForOverview.where)}
+                  />
+                </EuiFlexItem>
+              </EuiFlexGrid>
+            )}
+            <EuiSpacer size="s" />
+          </ContentPanel>
+          <EuiSpacer />
+          <FeaturePreview featureAttributes={context.queriesForOverview.feature_attributes} />
+          <EuiSpacer />
+        </EuiFlyoutBody>
+      </EuiPageBody>
+    ),
+    footerProps: {},
+    footer: (
+      <EuiFlyoutFooter>
+        {context.startedDetector ? null : (
+          <EuiFlexGroup alignItems="center" justifyContent="flexEnd">
+            <EuiFlexItem grow={false}>
+              <EuiButtonEmpty onClick={() => context.setFlyout(null)}>Cancel</EuiButtonEmpty>
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              <EuiButton
+                flush="right"
+                disabled={Object.keys(context.failures).length != 0}
+                onClick={() => createAndStartDetector(context)}
```

Comment:
If user click "Create Detector" button , will create detector first, and start to run this detector immediately ?

Reply:
Yes, I decided since the main goal behind this feature is to help users adopt AD and ease the set up process, it would be nice to automate the start step after creation from an already existing monitor.